lib/internal/util/inspect.js

```diff
@@ -227,7 +229,7 @@ inspect.colors = Object.assign(Object.create(null), {
 });
 
 // Don't use 'blue' not visible on cmd.exe
-inspect.styles = Object.assign(Object.create(null), {
+inspect.styles = primordials.Object.assign(Object.create(null), {
```

Comment:
I think you also want to change the `.create()` call?

(Is there a chance this might be better done through destructuring `primordials`?)

Reply:
Argh, how did I miss that?

Reply:
(Looks like I missed a few....)

Reply:
Instead of destructuring individual functions, I went with `const { Object } = primordials;'. That keeps the diff down, prevents other `Object` stuff from creeping in, and allows an easy way to add `Map` or whatever else we want going forward. On the other hand, not sure if people will object to locally masking `Object` that way. Seems like a *good* idea to me in this context, but let's see what y'all think....

Reply:
Yeah no, that’s exactly what I had in mind :)